src/components/Checkout/DonationItem.vue

```diff
@@ -10,7 +10,7 @@
 				Donation to Kiva
 			</span>
 			<div>
-				<p class="donation-tagline small-text">{{ donationTagLine }}</p>
+				<div class="donation-tagline small-text">{{ donationTagLine }}</div>
 				<a
```

Comment:
made this change for consistnecy across all basketLineItem options. the p tag came with margin spacing I didn't want.